package.json

```diff
@@ -75,18 +76,24 @@
     "babel-jest": "^23.4.0",
     "babel-plugin-transform-remove-console": "^6.9.4",
     "babel-preset-react-native": "^5.0.2",
+    "chalk": "^2.4.1",
```

Comment:
Could all of these (and `typescript` for that matter) be a devDependency?

Reply:
They are `devDependencies`, it just looks like they're dependencies here because of the fold on Github.